bofire/surrogates/trainable.py

```diff
@@ -96,6 +99,13 @@ def cross_validate(
             raise NotImplementedError(
                 "Cross validation not implemented for multi-output models"
             )
+
+        if stratified_feature is not None:
+            if stratified_feature not in experiments.columns:
```

Comment:
Good point. I'll make it check stratified_feature against input keys and output keys
Also the idea to check against the feature type is also good and should be easy to implement
I'll sort these out

From my point of view, have stratified CV with continuous features like how it is in this PR is not that useful. It would be more useful if we had a function that could sort the continuous feature into a certain number of bins according to the value and then perform stratified CV against the bin labels.

Reply:
But for now on, we can also let it work on every output type.